python/chronos/example/inference-acceleration/cpu_inference_acceleration.py

```diff
@@ -88,5 +88,6 @@ def predict_wraper(model, input_sample):
     speed_model = InferenceOptimizer.trace(lit_model, accelerator="onnxruntime", input_sample=input_sample)
 
     # evaluate the model's latency
+    torch.set_num_threads(1)
```

Comment:
could you update the readme Sample output as well?

Reply:
done